src/components/AboutContainer/AboutSideBar/AboutSideBar.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import useStyles from './AboutSideBar.styles';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import { Link } from '@material-ui/core';
+import { useHistory } from 'react-router-dom';
+import { PropTypes } from 'prop-types';
+import { useMediaQuery } from '@material-ui/core';
```

Comment:
Same with line 4 and 7. Can be combined

Reply:
Fixed in b57f2c7f4a31b6e2bfb46523d020db6754cdb21e